### PR TITLE
feat: request connector api, add local extension section

### PR DESCRIPTION
**Pull request by kyungeunni**
ticket: https://ubio-automation.atlassian.net/browse/ROBO-463
- [x] In API Connectors tab, add Request API Connector popup
  - [x] display it when showing `search` result
  - [x] Add a modal that creates the happy fox ticket
    - [x] need more information about happy fox integration => api keep returning 500 error

- [x] Local extension 
  - [x] move local extension within sections
  - [x] search bar works for both local and api-repository

image
image

### CODE REVIEW

File: packages/autopilot/src/app/viewports/extensions/extensions.vue [hunk 1]
```diff
@@ -159,9 +203,16 @@ export default {
...
 }
+
 .ext-list--empty {
     padding-bottom: var(--gap--large);
     font-size: var(--font-size);
 }
+
+.ext-dev-install {
+    padding: 0;
+    margin-top: -10px;
```
Comment (inca) [inline packages/autopilot/src/app/viewports/extensions/extensions.vue:216]: 🤨

File: packages/autopilot/src/app/viewports/extensions/request-connector.vue [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    flex-flow: row nowrap;
+    margin-bottom: var(--gap);
+    border-radius: var(--border-radius);
+    background: var(--color-cool--700);
+    color: #fff;
+    font-family: var(--font-family--alt);
+    font-size: var(--font-size--alt);
+}
+
+.rc__icon {
```
Comment (inca) [inline packages/autopilot/src/app/viewports/extensions/request-connector.vue:42]: Ah, here's our BEM issue again. I'm wary that this is a _yet another convention_ — but idk whether to change it or ignore it. Hope it gets better with CSS modules 🤞

**Review by inca — APPROVED** [review]
🎖️

File: packages/autopilot/src/app/viewports/extensions/request-connector.vue (hunk 2 above)
Comment (kyungeunni) [reply packages/autopilot/src/app/viewports/extensions/request-connector.vue:42]: You are right, I've updated them so `.rc__` are now `rc-`

**Pull request merged**